.github/ISSUE_TEMPLATE/10_bug_report.yml

```
@@ -13,7 +13,7 @@ body:
 - type: checkboxes
   attributes:
     label: Is there an existing issue for this?
-    description: Please search to see if an issue already exists for the bug you encountered (https://github.com/dotnet/aspnetcore/issues).
+    description: Please search to see if an issue already exists for the bug you encountered ([aspnetcore/issues](https://github.com/dotnet/aspnetcore/issues?q=is%3Aissue)).
```

> This works but I suggest contributors should be given more information about what to do when they find an existing issue, especially if it's closed. For example, should they try using a nightly build, reopen the closed issue, do something else❔

We already mention 

> More information on our issue management policies can be found here: https://aka.ms/aspnet/issue-policies

above, though I agree it isn't the first thing your eye is drawn to and may be missed. So how about adding it here as well:

```suggestion
    description: Please search to see if an issue already exists for the bug you encountered ([aspnetcore/issues](https://github.com/dotnet/aspnetcore/issues?q=is%3Aissue)). More information on our issue management policies is available [here](https://aka.ms/aspnet/issue-policies).
```


I don't believe we should be recommending nightlies as a blanket statement due to stability concerns and potential migration challenges they may have.

---

sounds good